src/ios/c_headers/coepicore.h

```diff
@@ -81,6 +81,8 @@ CFStringRef set_fever_taken_temperature_today(uint8_t c_is_set, uint8_t c_taken)
 
 CFStringRef set_symptom_ids(const char *c_ids);
 
+int32_t setup_logger(CoreLogLevel level, bool coepi_only);
```

Comment:
Params are log level and a flag stating whether only CoEpi logs should be shown.